### PR TITLE
[Integration][Dynatrace] Dynatrace bug causing SLO entities to be stale

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description

What - Fixed a bug where SLOs are returned unevaluated causing entities in Port to be stale
Why - 
How - Added an `evalute=true` query parameter to force evaluation

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
image

## API Documentation
[Get all SLOs](https://docs.dynatrace.com/docs/dynatrace-api/environment-api/service-level-objectives/get-all)

### CODE REVIEW

File: integrations/dynatrace/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.34 (2024-07-30)
+
+### Bug Fixes
+
+- Fix problem where API calls return unevaluated SLOs causing stale records in Port (0.1.34)
```
Comment (PeyGis) [inline integrations/dynatrace/CHANGELOG.md:14]:
```suggestion
- Fix problem where API calls return unevaluated SLOs causing stale records in Port
```

**Review by PeyGis — CHANGES_REQUESTED** [review]
Looks good, just one suggestion

**Review by matan84 — APPROVED** [review]
LGTM

**Review by PeyGis — APPROVED** [review]
LGTM

**Pull request merged**